### PR TITLE
Issue 214: convert body build doc to markdown format

### PR DESCRIPTION
**Pull request by apollokit**
https://github.com/nasa-jpl/open-source-rover/issues/214

### CODE REVIEW

File: Mechanical/Body Assembly/README.md [hunk 1]
```diff
@@ -19,22 +20,191 @@ The body is the housing of all the electronics for the rover. It is the attachme
...
+| Item | Ref | Qty | Image |
+| :--- | :-- | :-- | :---: |
+| Dual Side mount A            | S17 | 4 | <img src="../../images/components/Structural/S17.jpg" width="25%"> |
+| 4.5"x12" Aluminum Plate      | S37 | 2 | <img src="../../images/components/Structural/S37.jpg" width="40%"> |
+| 1" PVC Clamp                 | S24 | 1 | <img src="../../images/components/Structural/S24.jpg" width="70%"> |
+| 9"x12" Aluminum Plate        | S35A | 1 | <img src="../../images/components/Structural/S35.jpg" width="40%"> |
+| #6-32x1/4" Button Head Screw | B1 | 16 | <img src="../../images/components/Screws/B1.png" width="15%"> |
+| #6-32x3/8" Button Head Screw | B2 | 4 | <img src="../../images/components/Screws/B2.png" width="15%"> |
+| Allen Key Set                | | | <img src="../../images/components/Tools/D2.jpeg" width="30%"> |
+| 5/16" Wrench                 | | | <img src="../../images/components/Tools/D1.jpg" width="30%">  |
```
Comment (Achllle) [inline Mechanical/Body Assembly/README.md:82]: Can you split this into two columns to condense it?

File: Mechanical/Body Assembly/README.md [hunk 2]
```diff
@@ -19,22 +20,191 @@ The body is the housing of all the electronics for the rover. It is the attachme
...
+| :--- | :-- | :-- | :---: |
+| Dual Side Mount A              | S17 | 12 |  <img src="../../images/components/Structural/S17.jpg" width="15%"> |
+| #6-32x1/4" Button Head Screw   | B1 | 28 |   <img src="../../images/components/Screws/B1.png" width="8%"> |
+| 9"x12" Aluminum Plate          | S35 | 1 |   <img src="../../images/components/Structural/S35.jpg" width="30%"> |
+| Volt Meter                     | E38 | 1 |   <img src="../../images/components/Electronics/E38.png" width="30%"> |
+| Laser Cut Front Panel          | S39 | 1 |   <img src="../../images/components/Structural/S39.png" width="30%"> |
+| Laser Cut Back Panel           | S40 | 1 |   <img src="../../images/components/Structural/S40.png" width="30%"> |
+| Switch                         | E39 | 1 |   <img src="../../images/components/Electronics/E39.png" width="15%"> |
+| 3D printed Battery Holder      | S41 | 2 |   <img src="../../images/components/Structural/S41.png" width="30%"> |
+| #6-32 Heat set insert          | I2 | 4 |    <img src="../../images/components/Inserts/I2.png" width="15%"> |
```
Comment (Achllle) [inline Mechanical/Body Assembly/README.md:156]: Please split into two columns

**Review by Achllle — CHANGES_REQUESTED** [review]
Amazing! I made a few small comments:
* I dropped the section numbers in the [wheel build doc](https://github.com/nasa-jpl/open-source-rover/tree/master/Mechanical/Wheel%20Assembly) since Markdown doesn't autonumber. Whichever way we go, we should be consistent, so either update that doc or remove some of the numbering here.
* did you close any issues with this overhaul?
* tables waste a lot of space, would be good to split some of them up similar to how I did it in the wheel assy doc

File: Mechanical/Body Assembly/README.md (hunk 1 above)
Comment (apollokit) [reply Mechanical/Body Assembly/README.md:82]: Yeah, we'll split into two columns. It is pretty big as is.

**Comment by apollokit** [comment]
Needs update based on https://github.com/nasa-jpl/open-source-rover/pull/232

**Comment by apollokit** [comment]
Think it looks good now, ready for a final review

**Review by ericjunkins — APPROVED** [review]
It all looks good to me, if we want to take any of @Achllle 's suggestions those can still be discussed, but looks fine to me other than that.

**Review by Achllle — APPROVED** [review]
looks great!

**Pull request merged**